jsons/Policies.json

```diff
@@ -613,7 +613,7 @@
 			},
 			{
 				"name": "Media Culture",
-				"uniques": ["+[33]% [Culture] from every [Broadcast Tower]"],
+				"uniques": ["[+33]% Culture from every [Broadcast Tower]"],
```

Comment:
[Culture]

Reply:
resolved in https://github.com/ravignir/RekMOD/pull/43/commits/d98b1105a0e0d4f0661c3954f3b1811a2899ab7c